babbage/manager.py

```diff
@@ -70,3 +70,24 @@ def get_cube_model(self, name):
         file_name = os.path.join(self.directory, name + '.json')
         with open(file_name, 'r') as fh:
             return json.load(fh)
+
+
+class CachingJSONCubeManager(JSONCubeManager):
```

Comment:
we're not using this any more. I added it to babbage since I figured it might be useful and we needed it. We went further and now preload all cubes at startup since changing a model means redeploying anyway:
https://github.com/Code4SA/municipal-data/blob/a5416cf87bcebd8d8e52ca30bb408b46d9fc0677/municipal_finance/cubes.py#L9
https://github.com/Code4SA/municipal-data/blob/a5416cf87bcebd8d8e52ca30bb408b46d9fc0677/municipal_finance/__init__.py#L12